### PR TITLE
Add migration list command

### PR DESCRIPTION
**Pull request by Drab-Hounet**
add command in order to retrieve the list of all migrations created by the user
add unit tests

### CODE REVIEW

**Comment by cinlloc** [comment]
Thanks. Could you please check the CI error?

**Comment by lqueiroga** [comment]
CI error is normal. Tests around migration fail because uforge-python-sdk does not contains Migration objects

File: documentation/en/source/pages/command-line/migration.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.. Copyright (c) 2007-2017 UShareSoft, All rights reserved
+
+.. _command-line-migration:
+
+migration
+=========
+
+Manage all the migrations executed on live systems. The usage is:
```
Comment (lqueiroga) [inline documentation/en/source/pages/command-line/migration.rst]: I will say: Manage all the migrations launched by the user

File: documentation/en/source/pages/command-line/migration.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	usage: hammr migration [sub-command]
+
+
+Sub Commands
+------------
+
+``list`` sub-command
+~~~~~~~~~~~~~~~~~~~~
+
+Displays all the migration for the user.
```
Comment (lqueiroga) [inline documentation/en/source/pages/command-line/migration.rst]: migrationS

**Comment by lqueiroga** [comment]
Thanks for your PR

**Pull request merged**